go/chat/utils/utils.go

```diff
@@ -1080,10 +1080,12 @@ func presentAttachmentAssetInfo(ctx context.Context, g *globals.Context, msg cha
 			info.PreviewUrl = g.AttachmentURLSrv.GetURL(ctx, convID, msg.GetMessageID(), true)
 		}
 		atyp, err := asset.Metadata.AssetType()
-		if err == nil && atyp == chat1.AssetMetadataType_VIDEO && asset.Metadata.Video().DurationMs > 1 {
+		if err == nil && atyp == chat1.AssetMetadataType_VIDEO && asset.Metadata.Video().DurationMs > 1 &&
+			strings.HasPrefix(info.MimeType, "video") {
 			info.VideoDuration = new(string)
 			*info.VideoDuration = formatVideoDuration(asset.Metadata.Video().DurationMs) + ", " +
 				formatVideoSize(asset.Size)
+			info.InlineVideoPlayable = asset.Size < (1<<20)*500
```

Comment:
i think the shift operator is hard to understand at a glance, like better to just
```
1024 * 1024 * 500
```

Reply:
We use this all over the codebase.

Reply:
k